crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -6,6 +6,7 @@ pub mod deps {
     pub use serde;
     pub use serde_bytes;
     pub use serde_json;
+    pub use tracing;
```

Comment:
I backed out the code that's using this, but I'm going to come back to it -- bindgen needs to have tracing all over it (OTEL too!)

Reply:
I left a comment below just to point out where I see this need as well but I assume you're on it too 😄

Reply:
Yup, thanks! definitely need to pass that trace context around